bin/GameIndex.yaml

```diff
@@ -34227,7 +34227,7 @@ SLUS-20798:
   name: "Starcraft - Ghost"
   region: "NTSC-U"
 SLUS-20799:
-  name: "Terminator, The - Rise of the Machines"
+  name: "Terminator 3, The: Rise of the Machines"
```

Comment:
Not your mistake but there is no "The" on Terminator 3
https://i.ytimg.com/vi/Llp2QYdj8gk/maxresdefault.jpg